src/servers/ZoneServer2016/zoneserver.ts

```diff
@@ -5846,12 +5846,8 @@ export class ZoneServer2016 extends EventEmitter {
    */
   isArmor(itemDefinitionId: number): boolean {
     return (
-      this.getItemDefinition(itemDefinitionId)?.DESCRIPTION_ID ==
-        StringIds.TACTICAL_ARMOR ||
-      this.getItemDefinition(itemDefinitionId)?.DESCRIPTION_ID ==
-        StringIds.WOODEN_ARMOR ||
-      this.getItemDefinition(itemDefinitionId)?.DESCRIPTION_ID ==
-        StringIds.PLATED_ARMOR
+      this.getItemDefinition(itemDefinitionId)?.ITEM_CLASS ==
```

Comment:
Are only body armors using this item class? If not we might need to use the IS_ARMOR bool as well